Library/Formula/viewvc.rb

```diff
@@ -0,0 +1,35 @@
+class Viewvc < Formula
+  desc "Browser interface for CVS and Subversion repositories"
+  homepage "http://www.viewvc.org"
+  url "http://viewvc.tigris.org/files/documents/3330/49392/viewvc-1.1.23.tar.gz"
+  sha256 "d14290b23f44e8f5301a7ca9fc946bcd0f8bb8206d9b9c111ac56e19f2588b7d"
+
+  depends_on :python => :optional
+  if build.with? "python"
+    depends_on "subversion" => "with-python"
+  else
+    depends_on "subversion" => ["with-python", "universal"]
+  end
+
+  def install
+    system "./viewvc-install", "--prefix=#{libexec}", "--destdir="
+    (bin/"viewvc-standalone").write_env_script libexec/"bin/standalone.py",
```

Comment:
Can compress this into `bin.env_script_all_files(libexec, :PYTHONPATH => ENV["PYTHONPATH"])`